extensions/2.0/Khronos/KHR_materials_dispersion/README.md

```diff
@@ -0,0 +1,170 @@
+
+To calculate the index of refraction at a specific wavelength \( $\lambda$ \), given an Abbe number \( $V$ \) and the central index of refraction as specified by the ```KHR_materials_ior``` extension (assumed to be at the central wavelength, \( $N_d$ \)):
+
+$$
+B = \frac{n_d - 1}{V \left( {\lambda_F^{-2}} - {\lambda_C^{-2}} \right)}
+$$
+
+And then:
+
+$$
+A = n_d - \frac{B}{\lambda_d^2}
+$$
+
+And finally:
+
+$$
+n(\lambda) = A + \frac{B}{\lambda^2}
+$$
+
+![Dispersion on a Gem](./figures/Dispersion.jpg)
+
+In this extension, we store a transformed dispersion instead of the Abbe number directly.  Specifically we store $20/V$ so that a value of 1.0 is equivalent to $V=20$, which is about the lowest Abbe number for normal materials. Values over 1.0 are still valid for artists that want to exaggerate the effect. Decreasing values lower the amount of dispersion down to 0.0.  This is the same transform used by both Adobe Standard Material and OpenPBR.
+
+## Extending Materials
+
+The dispersion, defined in terms of Abbe number, is defined by adding the `KHR_materials_dispersion` extension to any glTF material.
+
+```json
+{
+    "materials": [
+        {
+            "extensions": {
+                "KHR_materials_dispersion": {
+                    "dispersion": 0.1
+                }
+            }
+        }
+    ]
+}
+```
+
+| |Type|Description|Required|
+|-|----|-----------|--------|
+| **dispersion** | `number` | The strength of the dispersion effect, specified as 20/Abbe number. | No, default: `0`|
+
+The default value of 0 has a special meaning in that no dispersion should be used.  This is the default value for backwards compatibility.  Any value zero or larger is considered to be a valid dispersion value, although the range between [0 , 1] is the range of realistic values.
+
+Here is a table of some material dispersion Abbe numbers, including the outlier of Rutile which is a very high dispersion material:
+
+| Material | Abbe Number (V) | Dispersion (20/V) |
+| -------- | ----------- | -------------- |
+| Rutile | 9.8 | 2.04 |
+| Polycarbonate | 32 | 0.625 |
+| Diamond | 55 | 0.36 |
+| Water | 55 | 0.36 |
+| Crown Glass | 59 | 0.33 |
+
+## Implementation
+
+*This section is non-normative.*
+
+One real-time method for rendering dispersion effects is to trace volume transmission separately for each of color channel accounting for the per channel IOR as determined by the Abbe number.  The resulting composite image will show color separation between the channels as a result.
+
+## Schema
+
+- [material.KHR_materials_dispersion.schema.json](schema/material.KHR_materials_dispersion.schema.json)
+
+## Reference
+
+### Theory, Documentation and Implementations
+
+[Abbe Number - Wikipedia](https://en.wikipedia.org/wiki/Abbe_number)
+
+[Abbe Number - Wolfram Formula Repository](https://resources.wolframcloud.com/FormulaRepository/resources/Abbe-Number#:~:text=The%20Abbe%20number%2C%20also%20known,of%20V%20indicating%20low%20dispersion.)
```

Comment:
Consider dropping the fragment part (starts with `#`) of the link.

Reply:
Fixed.